File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.java
class Solution {
    public int solution(int[] arr1, int[] arr2) {
        int arr1Length = arr1.length;
        int arr2Length = arr2.length;
        if (arr1Length > arr2Length) {
            return 1;
        } else if (arr1Length < arr2Length) {
            return -1;
        }
        
        int sum = 0;
        for (int i : arr1) {
            sum += i;
        }
        for (int i : arr2) {
            sum -= i;
        }
        return sum > 0 ? 1 : sum < 0 ? -1 : sum;
    }
}

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.py
def solution(arr1, arr2):
    if (len(arr1) != len(arr2)):
        return 1 if len(arr1) > len(arr2) else -1
    
    sumArr1 = sum(num for num in arr1)
    sumArr2 = sum(num for num in arr2)
    
    if arr1 == arr2 or sumArr1 == sumArr2:
        return 0
    
    if sumArr1 > sumArr2:
        return 1
    return -1